### PR TITLE
Add codelab ep lifecycle

### PR DESCRIPTION
**Pull request by jessemenning**

### CODE REVIEW

File: markdown/ep-lifecycle/ep-lifecycle.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+**For all sections, you'll need to:**
+- Complete the *Connecting Event Portal to a runtime Event Broker* code lab
+
+**For config push and CICD integration sections, you will also need to:**
+- Have a running Event Management Agent connected to an Event Broker
+- A GitHub account.  If you don’t have one, you can [get one free](https://github.com/signup). 
+
+## Discover event-driven stuff you didn’t know existed
+### Find interesting events to use in your solution
```
Comment (TamimiGitHub) [inline markdown/ep-lifecycle/ep-lifecycle.md:30]: We'll need a `Duration: 0:07:00` line under the  `## Discover event-driven stuff you didn’t know existed` title

File: markdown/ep-lifecycle/ep-lifecycle.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+1. In the home screen of Solace Cloud, click on Catalog.<br>![Image](img/1.png)<br><br>
+1. Within Catalog, click on the Events tab, then search for ```customer``` in the search box.  You’ll note that there are two existing customer events that would be perfect.<br>![Image](img/2.png)<br><br>
+1. Also, look for order related events by searching for ```order```<br>![Image](img/3.png)<br><br>
+
+## Easily design EDA solutions, together
```
Comment (TamimiGitHub) [inline markdown/ep-lifecycle/ep-lifecycle.md:35]: Add `Duration: 0:MM:00`

File: markdown/ep-lifecycle/ep-lifecycle.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+1. Name the queue ```C360.ORDERS``` (1)<br>
+give it a type of Solace Event Queue (2)<br>
+Click on Set Configuration (3) to use the default configuration
+Then click on Add Subscriptions (4) <br>![Image](img/15.png)<br><br>
+1. In the resulting screen, select the Order Created event, which will prefill the subscription.  Then click on Add Subscription<br>![Image](img/16.png)<br><br>
+1. Repeat the process to create the consumer for the two Customer events, naming the queue ```C360.CUSTOMERS```   When you click on Add Subscriptions, select both of the Customer events and a subscription for both.<br>![Image](img/17.png)<br><br>
+1. To save your changes, click on Save & Close.<br>![Image](img/18.png)<br><br>
+
+## Give your developers a headstart with code generation
+### AsyncAPI export
```
Comment (TamimiGitHub) [inline markdown/ep-lifecycle/ep-lifecycle.md:65]: Add `Duration: 0:MM:00`

File: markdown/ep-lifecycle/ep-lifecycle.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+1. In the resulting pop up, change the conversion to version 2.6.0, then click on Convert.<br>![Image](img/30.png)<br><br>
+1. Now that the AsyncAPI is loaded you can browse it, and also look through the autocreated documentation. 
+1. When you’re finished browsing, click again on the three dots (1), then click on Generate code/docs.(2)<br>![Image](img/31.png)<br><br>
+1. In the popup window, select Java Spring Cloud Stream Project, then click on Generate.
+1.  Save the resulting .zip file to your hard drive.  
+
+> aside positive
+>You’ll use the source code zip file in the CICD/GitHub Actions portion of the lab.
+## Provide exactly the events you need, when you need them
+### Assign your application to an environment/event broker
```
Comment (TamimiGitHub) [inline markdown/ep-lifecycle/ep-lifecycle.md:79]: Add `Duration: 0:MM:00`

File: markdown/ep-lifecycle/ep-lifecycle.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+### Remove application from runtime event broker and modelled event Mesh
+1. Click on the Customer360 link in the left hand column.<br>![Image](img/46.png)<br><br>
+1. In the resulting popup, click on Open in Designer.<br>![Image](img/47.png)<br><br>
+1. Click on the Runtime tab.<br>![Image](img/48.png)<br><br>
+1. Expand the Prod tab (1), click the three dots(2), the click on Remove Application.(3)<br>![Image](img/49.png)<br><br>
+1. In the popup, review the details of what queues will be removed.  Then click on Remove<br>![Image](img/50.png)<br><br>
+
+## Make your existing CICD even better
+> aside positive
```
Comment (TamimiGitHub) [inline markdown/ep-lifecycle/ep-lifecycle.md:109]: Add `Duration: 0:MM:00`

File: markdown/ep-lifecycle/ep-lifecycle.md [hunk 6]
```diff
@@ -0,0 +1,151 @@
...
+1. In the resulting window, change the radio button to Create a new branch for this commit.   Then click on Propose changes.<br>![Image](img/61.png)<br><br>
+1.  In the next screen, click on Create pull request.<br>![Image](img/62.png)<br><br>
+1. The GitHub Actions kick off.  When they are finished, there should be a list of changes that will occur should the pull request be approved.<br>![Image](img/63.png)<br><br>
+1.  Scroll down to the bottom of the pull request and click on Merge pull request.  Then click on Confirm merge.<br>![Image](img/64.png)<br><br>
+1.  Using the management console of your local broker, confirm that the CICD process created the same queues, subscriptions, ACLs as manual promotion.
+## Takeaways
+
+Duration: 0:07:00
+
+✅ < Fill IN TAKEAWAY 1>
```
Comment (TamimiGitHub) [inline markdown/ep-lifecycle/ep-lifecycle.md:145]: Update this section

File: markdown/ep-lifecycle/ep-lifecycle.md (hunk 1 above)
Comment (jessemenning) [reply markdown/ep-lifecycle/ep-lifecycle.md:33]: added

File: markdown/ep-lifecycle/ep-lifecycle.md (hunk 2 above)
Comment (jessemenning) [reply markdown/ep-lifecycle/ep-lifecycle.md:38]: added

File: markdown/ep-lifecycle/ep-lifecycle.md (hunk 4 above)
Comment (jessemenning) [reply markdown/ep-lifecycle/ep-lifecycle.md:109]: added

File: markdown/ep-lifecycle/ep-lifecycle.md (hunk 6 above)
Comment (jessemenning) [reply markdown/ep-lifecycle/ep-lifecycle.md]: added

**Review by TamimiGitHub — APPROVED** [review]
Looking hella hype 🔥 let's ship it

**Pull request merged**